sdk/src/DocuSign.eSign/Client/ApiClient.cs

```diff
@@ -407,9 +406,8 @@ public object Deserialize(IRestResponse response, Type type)
         /// </summary>
         /// <param name="content">Byte Araay (e.g. PDF bytes).</param>
         /// <param name="type">Object type.</param>
-        /// <param name="headers"></param>
         /// <returns>Object representation of the JSON string.</returns>
-        public object Deserialize(byte[] content, Type type, IList<Parameter> headers = null)
+        public object Deserialize(byte[] content, Type type)
```

Comment:
@RajRele this is a public method. Hopefully no one is using it otherwise their code might need to be updated

Reply:
https://github.com/docusign/docusign-esign-csharp-client/pull/338#discussion_r675000444